provisioning/miniprov/hare_mp/main.py

```diff
@@ -287,11 +287,23 @@ def save(filename: str, contents: str) -> None:
         f.write(contents)
 
 
+def generate_config(path_to_cdf: str):
```

Comment:
I think if we are creating this function, we need to use it at other places too wherever config is generated using CDF.
What do you think @mssawant and @knekrasov ?
Thanks!

Reply:
I wouldn't say it makes much sense in this case. The fact is that the current file doesn't have any other places where cfgen is executed like this. There is another one - in CdfGenerator class. If we want to refactor cfgen invocation so that both this main.py file and CdfGenerator reuse the same piece of code, then cfgen invocation should probably be refactored into a separate class (I don't like static functions, so let's don't consider that option) e.g. CfgenRunner or something. Anyway, I don't see much benefit. Potentially, it can be helpful if we want to test our business logic without real cfgen (e.g. we want to mock cfgen and emulate its behavior in a predictable and repeatable way). But at this moment we don't have such a need.

We don't save that much lines of code but require extra efforts.